src/nerve/server/engine.py

```
@@ -1307,6 +1357,12 @@ async def _cleanup_on_stop(self) -> None:
             except Exception:
                 pass  # Best effort cleanup
 
+        # Stop all proxies
+        try:
+            await self._proxy_manager.stop_all()
+        except Exception:
+            pass  # Best effort cleanup
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)